pegasus/sites.v3/code.org/public/ai/pl/101/index.haml

```diff
@@ -26,98 +26,79 @@ theme: responsive_full_width
             =hoc_s(:ai_pl_101_explore__button_3)
 
       .action-block.action-block--explore.action-block--one-col.flex-space-between
         .content-wrapper
           %p.overline=hoc_s(:module_label_student_tool)
+          %img{src: "/images/ai/pl/101/explore_4.png", alt: ""}
           %h3=hoc_s(:ai_pl_101_explore_title_4)
-          %img{src: "/images/ai/pl/101/explore_4.jpg", alt: ""}
           %p=hoc_s(:ai_pl_101_explore__desc_4)
         .content-footer
-          %a{href: "https://www.ets.org/research/ai-labs.html", target: "_blank", class: "link-button",  rel: "noopener noreferrer", aria:{label: hoc_s(:ai_resources_btn_aria_label_01)}}
+          %a.link-button{href: "https://www.ets.org/research/ai-labs.html", target: "_blank", rel: "noopener noreferrer", aria:{label: hoc_s(:ai_resources_btn_aria_label_01)}}
             =hoc_s(:ai_pl_101_explore__button_4)
 
       .action-block.action-block--explore.action-block--one-col.flex-space-between
         .content-wrapper
           %p.overline=hoc_s(:module_label_educator_course)
+          %img{src: "/images/ai/pl/101/explore_2.png", alt: ""}
           %h3=hoc_s(:ai_pl_101_explore_title_2)
-          %img{src: "/images/ai/pl/101/explore_2.jpg", alt: ""}
           %p=hoc_s(:ai_pl_101_explore__desc_2)
         .content-footer
-          %a{href: "https://www.iste.org/professional-development/iste-u/artificial-intelligence", target: "_blank", class: "link-button",  rel: "noopener noreferrer", aria:{label: hoc_s(:ai_resources_btn_aria_label_01)}}
+          %a.link-button{href: "https://www.iste.org/professional-development/iste-u/artificial-intelligence", target: "_blank", rel: "noopener noreferrer", aria:{label: hoc_s(:ai_resources_btn_aria_label_01)}}
             =hoc_s(:ai_pl_101_explore__button_2)
 
       .action-block.action-block--explore.action-block--one-col.flex-space-between
         .content-wrapper
           %p.overline=hoc_s(:module_label_educator_tool)
+          %img{src: "/images/ai/pl/101/explore_1.png", alt: ""}
           %h3=hoc_s(:ai_pl_101_explore_title_1)
-          %span.action-block__space
-          %img{src: "/images/ai/pl/101/explore_1.jpg", alt: ""}
           %p=hoc_s(:ai_pl_101_explore__desc_1)
         .content-footer
-          %a{href: "https://blog.khanacademy.org/teacher-khanmigo/?utm_source=code.org&utm_medium=referral&utm_campaign=code.orgwebinar", target: "_blank", class: "link-button",  rel: "noopener noreferrer", aria:{label: hoc_s(:ai_resources_btn_aria_label_01)}}
+          %a.link-button{href: "https://blog.khanacademy.org/teacher-khanmigo/?utm_source=code.org&utm_medium=referral&utm_campaign=code.orgwebinar", target: "_blank", rel: "noopener noreferrer", aria:{label: hoc_s(:ai_resources_btn_aria_label_01)}}
             =hoc_s(:ai_pl_101_explore__button_1)
 
+= view :visit_partial_event_logger
```

Comment:
For this event to work, I think it will need an addition to `visit_partial_event_logger.haml` to map '/ai/pl/101' to it's associated event (probably should be listed above the '/ai' one so that when it's matching strings it sees '/ai/pl/101' before '/ai' since '/ai/pl/101' contains '/ai'

Reply:
@TurnerRiley good catch! I just added that, and put the `visit_partial_event_logger` view on code.org/ai page too, in [this commit](https://github.com/code-dot-org/code-dot-org/pull/53214/commits/ea7cc766a386d25e86e97aa18c3ec93b64ef895b) — can you double check this is correct?

Reply:
Looks perfect!